modules/manual-configuration-image-registry-qcr.adoc

```diff
@@ -5,21 +5,35 @@
 [id="manual-configuration-image-registry-qcr_{context}"]
 = Manually configuring Quay Container Registry
 
-You can integrate {product-title} with Quay Container Registry.
+You can integrate {product-title} ({product-title-short}) with Quay Container Registry. You can integrate with Quay using the following methods:
+
+- Integrating with the Quay public repository (registry): This method does not require authentication.
+- Integrating with a Quay private registry using a robot account: This method requires that you create a robot account to use with Quay (recommended). See the link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/use_red_hat_quay/use-quay-manage-repo#allow-robot-access-user-repo[Quay documentation] for more information.
+- Integrating with Quay to use the Quay scanner rather than the {product-title-short} scanner:  This method uses the API and requires an OAuth token for authentication.
```

Comment:
This section is about registries not scanner. There is a separate page for scanner in the docs: https://docs.openshift.com/acs/3.71/integration/integrate-with-image-vulnerability-scanners.html#integrate-with-qcr-scanner_integrate-with-image-vulnerability-scanners

I think for consistency's sake it's best to keep it split (even though in the UI it's not split).

Reply:
Oh, my fault, I didn't realize this. I'll just link to the existing doc (even though links in our templates are...unwieldy).